README.md

```diff
@@ -1,56 +1,60 @@
-# NodeIngestBench
+# Node.js CrateDB Ingest Benchmark
 
-a simple nodejs script to quickly test ingest performance on CrateDB clusters.
+A multi-process Node.js script to run high-performance ingest benchmarks on CrateDB clusters.
+The script generates random data and runs batched insert statements against a single table using CrateDB's [HTTP endpoint](https://crate.io/docs/crate/reference/en/4.7/interfaces/http.html).
 
-top measured performance with single nodejs client : 1 200 000 rows / s
+The top measured performance with a single Node.js process was 1,200,000 rows/s.
```

Comment:
@proddata do you recall some more details of this measurement (how many nodes)?

Reply:
7 nodes à 30 vCPUs, but that was also before any optimization